content/reference/compose-file/services.md

```diff
@@ -1505,6 +1505,35 @@ ports:
     mode: host
 ```
 
+## post_start
+
+`post_start` defines a sequence of lifecycle hooks to run after a container has started. The exact timing of when the command is run is not guaranteed.
+
+- `command`: The command to run after the container has started. This attribute is required.
```

Comment:
Probably worth mentioning that this works the same way as `CMD` and the default `command` (E.g., you can use both the shell form and exec form. In the case of exec form, no shell is invoked.)